app/controllers/mixins/generic_session_mixin.rb

```diff
@@ -4,24 +4,34 @@ module GenericSessionMixin
     private
 
     def get_session_data
-      prefix      = self.class.session_key_prefix
-      @title      = respond_to?(:title) ? title : ui_lookup(:tables => self.class.table_name)
-      @layout     = prefix
-      @table_name = request.parameters[:controller]
-      @lastaction = session["#{prefix}_lastaction".to_sym]
-      @display    = session["#{prefix}_display".to_sym]
-      @filters    = session["#{prefix}_filters".to_sym]
-      @catinfo    = session["#{prefix}_catinfo".to_sym]
-      @showtype   = session["#{prefix}_showtype".to_sym]
+      prefix          = self.class.session_key_prefix
+      @title          = respond_to?(:title) ? title : ui_lookup(:tables => self.class.table_name)
+      @layout         = prefix
+      @table_name     = request.parameters[:controller]
+      @lastaction     = session["#{prefix}_lastaction".to_sym]
+      @display        = session["#{prefix}_display".to_sym]
+      @filters        = session["#{prefix}_filters".to_sym]
+      @catinfo        = session["#{prefix}_catinfo".to_sym]
+      @cats           = session["#{prefix}_cats".to_sym]
+      @showtype       = session["#{prefix}_showtype".to_sym]
+      @base           = session["#{prefix}_compare_base".to_sym]
```

Comment:
OK, the same goes for `@base` .. https://github.com/ManageIQ/manageiq-ui-classic/pull/665